src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,50 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("Starting the process of completing an order for userId: {}", userId);
```

Comment:
The log message should be more informative. Include the method name and the user ID to make it clear which operation is starting. For example, 'Starting the process of completing an order in completeOrder method for userId: {}'